### PR TITLE
Curating Owners: pkg/kubectl

### PR DESCRIPTION
**Pull request by apelisse**
In an effort to expand the existing pool of reviewers and establish a
two-tiered review process (first someone lgtms and then someone
experienced in the project approves), we are adding new reviewers to
existing owners files.
## If You Care About the Process:

We did this by algorithmically figuring out who’s contributed code to
the project and in what directories.  Unfortunately, that doesn’t work
well: people that have made mechanical code changes (e.g change the
copyright header across all directories) end up as reviewers in lots of
places.

Instead of using pure commit data, we generated an excessively large
list of reviewers and pruned based on all time commit data, recent
commit data and review data (number of PRs commented on).

At this point we have a decent list of reviewers, but it needs one last
pass for fine tuning.
## TLDR:

As an owner of a sig/directory and a leader of the project, here’s what
we need from you:
1. Comment on the pull-request explaining the changes you want. I'll
   take care of doing them (unless you want to modify the pull-request
   yourself).
2. OWNERS files in subdirectory can be removed if they are redundant with
   top-directory files.
3. Please use the ALIAS feature if you want to re-use a list of persons
   in different files.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35715)

### CODE REVIEW

**Comment by pwittrock** [comment]
@smarterclayton @fabianofranz @AdoHe @brendandburns @deads2k

Would you all take a look at this PR to see if it seems correct to you?  You can compare the original suggested list vs my pruning in the commit history of the PR.  Please let me know if you think that I should remove anyone else, or add anyone back in that I removed.

**Comment by smarterclayton** [comment]
Is @janetkuo still involved in kubectl?  She would probably be on my owners list expectations if she is still planning on being active.

**Comment by pwittrock** [comment]
@janetkuo AFAIK Janet isn't actively working on kubectl and is focussed more on workloads.  @janetkuo wdyt?

**Comment by janetkuo** [comment]
Yes I'm now focusing more on workloads. I don't mind being on the approver/reviewer list if needed

**Comment by pwittrock** [comment]
@janetkuo Glad to add you back then :)

**Comment by AdoHe** [comment]
LGTM.

**Comment by deads2k** [comment]
lgtm

**Comment by fabianofranz** [comment]
+1 LGTM

**Comment by pwittrock** [comment]
Cool @apelisse is going to dedup this and pull it into an alias' file

**Comment by pwittrock** [comment]
@brendandburns LMK if you would prefer to be taken out of approvers or stay in it.  I will leave you in until I hear back from you.

**Comment by apelisse** [comment]
_Adding **do-not-merge**_

I need to triple-check that this new format is activated before this get merged. I'll remove after.

**Comment by apelisse** [comment]
@k8s-bot cvm gke e2e test this

**Comment by apelisse** [comment]
@k8s-bot cvm gce e2e test this

**Comment by apelisse** [comment]
Let's try to get this merged and see what happens.

**Comment by spxtr** [comment]
Go home @k8s-merge-robot, you're drunk.

(The problem is that we have a backlog of Jenkins jobs that is longer than the 30 minutes period that the munger runs on.)

**Comment by apelisse** [comment]
I can drive him home if needed ...
